pandera/backends/pandas/__init__.py

```diff
@@ -2,7 +2,8 @@
 
 import pandas as pd
 
-import pandera.typing
+# import pandera.typing
+from pandera.typing import *
```

Comment:
explicit imports are preferred over `*` imports, as it better controls what symbols are imported into the namespace.